podcasts/Beta/BetaMenu.swift

```diff
@@ -0,0 +1,30 @@
+import SwiftUI
+
+struct BetaMenu: View {
+    var body: some View {
+        List {
+            ForEach(FeatureFlag.allCases, id: \.self) { feature in
+                Toggle(feature.rawValue, isOn: feature.isOn)
```

Comment:
Since the `OverrideableFlag` conforms to `CustomStringConvertible` we can change from the rawValue to make sure it uses the description when we add them.

```suggestion
                Toggle(String(describing: feature), isOn: feature.isOn)
```